build/arm.sh

```diff
@@ -93,16 +93,31 @@ EOF
 
 mkdir -p ${STAGEDIR}/boot/msdos
 mount_msdosfs /dev/${DEV}s1 ${STAGEDIR}/boot/msdos
-
 mkdir -p ${STAGEDIR}/boot/msdos/overlays
-cp -p ${STAGEDIR}/boot/ubldr ${STAGEDIR}/boot/msdos/ubldr
-cp -p ${STAGEDIR}/boot/ubldr.bin ${STAGEDIR}/boot/msdos/ubldr.bin
+
+case "${PRODUCT_ARCH}" in
+armv6)
+	cp -p ${STAGEDIR}/boot/ubldr ${STAGEDIR}/boot/msdos/ubldr
+	cp -p ${STAGEDIR}/boot/ubldr.bin ${STAGEDIR}/boot/msdos/ubldr.bin
+	;;
+aarch64)
+	cp -p ${STAGEDIR}/boot/loader.efi ${STAGEDIR}/boot/msdos/bootaa64.efi
+	;;
+esac
 
 case "${PRODUCT_DEVICE}" in
 bpi)
 	cp -p ${STAGEDIR}/boot/dtb/bananapi.dtb ${STAGEDIR}/boot/msdos/sun7i-a20-bananapi.dtb
 	cp -p /usr/local/share/u-boot/u-boot-bananapi/* ${STAGEDIR}/boot/msdos
 	;;
+nanopi-neo2
+	cp -p ${STAGEDIR}/boot/dtb/sun50i-h5-nanopi-neo2.dtb ${STAGEDIR}/boot/msdos/sun50i-h5-nanopi-neo2.dtb
+	cp -p /usr/local/share/u-boot/u-boot-nanopi-neo2/* ${STAGEDIR}/boot/msdos
+	;;
+nanopi-r1
```

Comment:
can you rename this one to nanopi-neo like in u-boot?

Reply:
Sure, hope that NEO image will be compatible with NanoPi R1.